extensions/interactions/InteractiveMap/InteractiveMap.py

```diff
@@ -27,6 +27,7 @@ class InteractiveMap(base.BaseInteraction):
     _dependency_ids = ['google_maps']
     answer_type = 'CoordTwoDim'
     instructions = 'Click on the map'
+    narrow_instructions = 'Click to view map'
```

Comment:
Edit: probably "View clickable map" is better (to indicate that the map can be clicked)